context-free.md

```diff
@@ -2,21 +2,28 @@
 
 ##  Who is this programming language for?
 
+Anyone who is interested in creating procedurally generated art.
 
 ## What is easy to do in this language? Why is it easy?
 
+Creating self similar images is very straight forward due to the way the rules call each other.
 
 ## What is hard to do in this language? Why is it hard?
 
+Controlling randomness seems a bit hard. It would be nice to do something other than just
+trying a bunch of different seeds to see what works.
 
 ## How did you learn how to program in this language?
 
+By looking at examples, both online and other students' submissions.
+
 
 ## What is the underlying _computational model_ for this programming language? 
-_We don't yet have a great definition of the term "computational model". 
-For now, try to come up with the clearest, most concise explanation of what 
-happens when a ContextFree program runs._
 
+I presume the program gets parsed and a grammar built. Then the grammar gets effectively
+enumerated following each rule to produce the picture.
 
 ## What do you think is interesting about the ContextFree program you wrote?
 
+I was trying to make a Dragon Curve, but I found that if you didn't reverse the fold
```

Comment:
This looks really cool! I like how you used path declarations (which I didn't use in my program, but it would've made my life easier coding my image).